tests/system/env-template.yml

```diff
@@ -100,8 +100,17 @@ mlrun_model_monitoring_stream_profile:
   # type: kafka_source
   # brokers: # <host>:<port>
   # topics: []
+  # With SASL:
   # sasl_user:
   # sasl_pass:
+  # For Confluent Kafka:
+  # kwargs_public:
+  #   security_protocol: SASL_SSL
+  #   api_version_auto_timeout_ms: 20000
+  #   tls:
+  #     enable: true
+  #   new_topic:
+  #     replication_factor: 3
```

Comment:
I guess it should also mention `num_partitions`? As in:
```suggestion
  #     replication_factor: 3
  #     num_partitions: 8
```

Reply:
The replication factor is specified since it's locked in Confluent Cloud (at least in the "basic" and "standard" cluster types):
![image](https://github.com/user-attachments/assets/19406b8c-a352-490b-a6bf-9807e2953d33)
So our default value for creating new topics clashes with it:
https://github.com/mlrun/mlrun/blob/eddb535be1779f75123b42827a4db011d89bcfa4/mlrun/config.py#L569-L571

This is not the case for the number of Kafka partitions, which can be adjusted:
![image](https://github.com/user-attachments/assets/36470970-f5a1-406d-8506-fe30281c0e09)